### PR TITLE
Add change log and bump version

### PR DESCRIPTION
**Pull request by Flanker32**
Bump web app to `1.7.0` and update the release note.

### CODE REVIEW

File: azure-functions-maven-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
   - [1.3.2](#132)
   - [1.3.1](#131)
   - [1.3.0](#130)
   - [1.2.2](#122)
   - [1.2.1](#121)
   - [1.2.0](#120)
 
+## 1.3.3
+- Set `FUNCTIONS_WORKER_RUNTIME` to `java` by default.[#400](https://github.com/microsoft/azure-maven-plugins/issues/400)
+- Exit when no annotated methods was found in project folder.[#426](https://github.com/microsoft/azure-maven-plugins/issues/426)
```
Comment (andxu) [inline azure-functions-maven-plugin/CHANGELOG.md]: was -> are

File: azure-functions-maven-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
+- Fix plugin can't get correct subscription in azure cloud shell.[#628](https://github.com/microsoft/azure-maven-plugins/issues/628)
```
Comment (andxu) [inline azure-functions-maven-plugin/CHANGELOG.md]: Fix: Plugin

Comment (Flanker32) [reply azure-functions-maven-plugin/CHANGELOG.md]: Fixed

File: azure-functions-maven-plugin/CHANGELOG.md (hunk 1 above)
Comment (Flanker32) [reply azure-functions-maven-plugin/CHANGELOG.md]: Fixed

**Pull request merged**